src/controllers/scripting/legacy/controllerscriptinterface.cpp

```diff
@@ -0,0 +1,872 @@
+#include "controllerscriptinterface.h"
+
+#include "control/controlobject.h"
+#include "control/controlobjectscript.h"
+#include "controllers/controllerdebug.h"
+#include "controllers/scripting/legacy/controllerscriptenginelegacy.h"
+#include "controllers/scripting/legacy/scriptconnectionjsproxy.h"
+#include "mixer/playermanager.h"
+#include "util/math.h"
+#include "util/time.h"
+
+namespace {
+const int kDecks = 16;
+
+// Use 1ms for the Alpha-Beta dt. We're assuming the OS actually gives us a 1ms
+// timer.
+const int kScratchTimerMs = 1;
+const double kAlphaBetaDt = kScratchTimerMs / 1000.0;
+} // anonymous namespace
+
+ControllerScriptInterface::ControllerScriptInterface(ControllerScriptEngineLegacy* m_pEngine)
```

Comment:
We should put Legacy into all class names or make a dedicated `mixxx::controllers::legacy` namespace or something like that.

Reply:
Okay. I hesitated to do that because "ControllerScriptInterfaceLegacy" is quite a long class name.

Reply:
done in ea63abe19308d6721c493691b2a9e12474ada8ab